File: src/test/java/net/questcraft/apitests/SecondMixinTestCase.java
package net.questcraft.apitests;

import net.yakclient.mixin.api.Injection;
import net.yakclient.mixin.api.InjectionType;
import net.yakclient.mixin.api.Mixer;
import org.objectweb.asm.Opcodes;

@Mixer("net.questcraft.apitests.MixinSourceClassTest")
public abstract class SecondMixinTestCase {
    @Injection
    public void printTheString(int integer) {
        System.out.println("I also wanna print this");
    }
}

File: base/src/test/kotlin/net/yakclient/example/SecondMixinTestCase.kt
package net.yakclient.example

import net.yakclient.mixin.api.Injection
import net.yakclient.mixin.api.InjectionType
import net.yakclient.mixin.api.Mixer

@Mixer("net.yakclient.example.MixinSourceClassTest")
abstract class SecondMixinTestCase {
    @Injection(type = InjectionType.BEFORE_RETURN)
    fun printTheString() {
        println("I also wanna print this")
    }
}